ReactWindows/ReactNative/Views/Picker/ReactPickerManager.cs

```diff
@@ -0,0 +1,210 @@
+﻿using Newtonsoft.Json.Linq;
+using ReactNative.UIManager;
+using ReactNative.UIManager.Events;
+using System;
+using Windows.UI;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Media;
+
+namespace ReactNative.Views.Picker
+{
+    /// <summary>
+    /// A view manager responsible for rendering picker.
+    /// </summary>
+    public class ReactPickerManager : BaseViewManager<ComboBox, ReactPickerShadowNode>
+    {
+        private const string PROP_COLOR     = "color";

```

Comment:
In the other modules, we haven't really been using consts for property names.  The C# compiler will generate consts where it can, and this just adds an unecessary indirection step to figure out what the property name is.